src/browser/base/content/ZenUIManager.mjs

```
@@ -193,25 +202,39 @@
-  //  } else {
-  //    button.removeAttribute('open');
-  //  }
-  //},
-
   toggleExpand() {
-    let expanded = !this.expanded;
-    Services.prefs.setBoolPref('zen.view.sidebar-expanded', expanded);
-    Services.prefs.setBoolPref('zen.view.sidebar-expanded.on-hover', false);
+    const pausedForExpand = Services.prefs.getBoolPref('zen.view.sidebar-expanded.on-hover.paused-for-expand');
+    const onHover = Services.prefs.getBoolPref('zen.view.sidebar-expanded.on-hover');
+    let expanded = Services.prefs.getBoolPref('zen.view.sidebar-expanded');
```

Biome

[error] This let declares a variable that is only assigned once.

'expanded' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)